tests/test_numseq.py

```diff
@@ -45,10 +33,45 @@ def test_geo(self):
         """varyifying geo functions from range 0 to 9"""
```

Comment:
The summary line of docstrings should be capitalized and end with a period.
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings